newIDE/app/src/MainFrame/Preferences/PreferencesProvider.js

```diff
@@ -447,6 +449,33 @@ export default class PreferencesProvider extends React.Component<Props, State> {
     );
   }
 
+  _showAnnouncement(identifier: string, show: boolean) {
```

Comment:
looking at the other props in the preferences, it's probably too late now, but I find the choice of name a bit confusing - especially if the only action available is hiding one.

Reply:
I agree but I'll favor the consistency with the rest for now!